pkg/adapter/adapter.go

```diff
@@ -452,15 +475,16 @@ func (a *Adapter) ExecuteTxs(
 		PartSetHeader: cmttypes.PartSetHeader{Total: 1, Hash: block.DataHash},
 	}
 
-	if err := fireEvents(a.EventBus, block, currentBlockID, fbResp, validatorUpdates); err != nil {
-		a.Logger.Error("failed to fire events", "err", err)
-	}
-
 	// save the finalized block response
 	if err := a.Store.SaveBlockResponse(ctx, blockHeight, fbResp); err != nil {
```

Comment:
same comment as yesterday about possibly extracting the events from here as well and add them back at the signed epoch block